be/src/common/config.cpp

```diff
@@ -998,6 +998,8 @@ DEFINE_mInt32(s3_write_buffer_whole_size, "524288000");
 //disable shrink memory by default
 DEFINE_Bool(enable_shrink_memory, "false");
 
+DEFINE_Bool(enable_set_in_bitmap_value, "true");
```

Comment:
we need add more comment about enable_set_in_bitmap_value and SET_TYPE_THRESHOLD.

Reply:
we need pick it to 1.2.5 and set its default value to false. And then in 2.x we need set it to true to remove the if-else. Because it will change the storage format. @cambyzju